srcpkgs/osquery-voidlinux/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'osquery-voidlinux'
+pkgname=osquery-voidlinux
+version=0.1.0
+revision=1
+build_style=cargo
+short_desc="Extension for osquery to provide Void Linux specific information"
+maintainer="Jan Christian Grünhage <jan.christian@gruenhage.xyz>"
+license="AGPL-3.0-or-later"
+homepage="https://git.jcg.re/jcgruenhage/osquery-voidlinux"
+changelog="https://git.jcg.re/jcgruenhage/osquery-voidlinux/raw/branch/main/CHANGELOG.md"
+distfiles="https://git.jcg.re/jcgruenhage/osquery-voidlinux/archive/v${version}.tar.gz"
+checksum=1608d0932711c25361267975130000e0d51a605a297cab8006af4b8a7854d922
+
+do_install() {
+	vbin target/${RUST_TARGET}/release/osquery-voidlinux osquery-voidlinux.ext
```

Comment:
I'm not quite sure whether this is the right place actually, https://osquery.readthedocs.io/en/stable/deployment/extensions/ sounds like `/usr/lib/osquery/extensions/<extension>.ext` would be the right place